app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict[str, str | int]]) -> list[Person]:
+    person_list = []
+
+    # Перший цикл: створення екземплярів
+    for person_dict in people:
+        name = person_dict["name"]
+        age = person_dict["age"]
+        person = Person(name, age)
+        person_list.append(person)
```

Comment:
use list comprehension here

Reply:
Thanks for the advice! I'll add it in the next commit.